src/profile/Languages.js

```diff
@@ -0,0 +1,137 @@
+import React, {useState, useEffect} from "react";
+import { useHistory } from 'react-router-dom';
+import FormGroup from "react-bootstrap/lib/FormGroup";
+import ControlLabel from "react-bootstrap/lib/ControlLabel";
+import FormControl from "react-bootstrap/lib/FormControl";
+import LoaderButton from "../components/LoaderButton";
+import "react-quill/dist/quill.snow.css";
+import Button from "react-bootstrap/lib/Button";
+import Tour from "reactour";
+import { I18n, USER_AGENT_HEADER } from "@aws-amplify/core";
+import API from "@aws-amplify/api";
+
+
+const Languages = (props) => {
+  let [state, setState] = useState({defaultLanguage:""});
+  let [isLoading, setIsLoading] = useState(false);
+  let [isTourOpen, setIsTourOpen] = useState(false);
+  let [defaultLanguage, setDefaultLanguage] = useState("");
+  
+
+async function fetchUser(id){
+  const response = await API.get("pareto", `/users/${id}`);
+  return response;
+}
+
+useEffect ( () => {
+    const path = window.location.pathname.split("/");
+    let userId = path[path.length - 1]; 
+    // what we did above, was the get the user id from the navigation bar
+    let user = fetchUser(userId);
+    setState(user);
+
+},[])
+
+   // This function below handles the changes in state, based on the forms. All of the information stored in the forms, is stored in state. Each form has an `id`, which is accessed by the event.target.id.
+  // The actual updated value, is represented by the event.target.value. I recommend you console.log both of the values, above the setState, so you understand.
+
+const handleChange = (e) => {
+    console.log("value: ",e.target.value);
```

Comment:
Nit: remove console.log's from PR's.

FYI A 'nit' is a small comment, common dev speak.